nss-day-cohort-60/national-parks-django-api
# added images to campgrounds

Description:
1. Added image field to campground model
2. Added images to each campground in national_park_add_data.sql
3. Updated serializer in campground_view.py

Steps to test:
1. Fetch md-campground-images and switch to it
2. Delete your database and migration files
3. Run the below commands in terminal:
     python3 manage.py makemigrations parksapi
     python3 manage.py migrate
     python3 manage.py loaddata users
     python3 manage.py loaddata tokens
Then re-run the national_park_add_data.sql queries
To run, highlight all text in the file and use cmd + R + R
Check the database and ensure the values have been updated
